tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -164,7 +164,7 @@ def setup_system(self, start_processor=True, num_threads=3):
 
         # Start test harness
         self.driver = StreamsSmokeTestDriverService(self.test_context, self.kafka)
-        self.processor1 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka, num_threads)
+        self.processor1 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka, "at_least_once", num_threads)
```

Comment:
Is it enough to specify the processing guarantee as `at_least_once` here or do you want also for this test to include all processing guarantees in the test matrix?

Reply:
On a different note, now that the processing guarantee can be passed to the service, do we still need `StreamsEosTestJobRunnerService` and `StreamsComplexEosTestJobRunnerService`?

Reply:
I feel neutral about it, if we are adding 2X time to system tests, that's probably not good.

Reply:
We can try to add both eos cases, too.

For `Streams[Complex]EosTestJobRunnerService`, they use a different Java class compared to the "smoke test" runner. Not sure if we could unify them?